fmn/rules/utils.py

```diff
@@ -67,6 +72,32 @@ def get_fas(config):
     return _FAS
 
 
+def _paginate_pagure_data(url, params, key):
```

Comment:
Bodhi does have some code from @mprahl to make API calls to Pagure - I would love to use an official Python binding library if such a thing were to be created. It's not a huge deal though, as Bodhi only really makes "light" usage of Pagure so far.

https://github.com/fedora-infra/bodhi/blob/2.8.1/bodhi/server/util.py#L775-L781